packages/babel-preset-default/package.json

```diff
@@ -39,9 +40,12 @@
 		"@wordpress/element": "file:../element",
 		"@wordpress/warning": "file:../warning",
 		"browserslist": "^4.16.6",
-		"core-js": "^3.6.4"
+		"core-js": "^3.12.1"
```

Comment:
This seems to be a different version than the top-level `package.json` and `react-native-editor`. Is that intentional?

Reply:
I will align, I might miss some places because I updated it during rebase when I saw a confict 😅